zip-0000.rst

```diff
@@ -0,0 +1,578 @@
+
+If there are multiple champions, each should be on a separate line.
+
+While a ZIP is in private discussions (usually during the initial Draft
+phase), a Discussions-To header will indicate the mailing list or URL
+where the ZIP is being discussed. No Discussions-To header is necessary
+if the ZIP is being discussed privately with the Champion, or on the
+Zcash email mailing lists.
+
+The Category header specifies the type of ZIP: Consensus, Standards Track,
+Informational, or Process.
+
+The Created header records the date that the ZIP was submitted.
+Dates should be in yyyy-mm-dd format, e.g. 2001-08-14.
+
+Auxiliary Files
+---------------
+
+ZIPs may include auxiliary files such as diagrams. Auxiliary files
+should be included in a subdirectory for that ZIP; that is, any ZIP that
+requires more than one file should be in a subdirectory named zip-XXXX.
+
+
+ZIP types
+=========
+
+There are several kinds of ZIP:
+
+* A Consensus ZIP describes a change that affects the consensus protocol
+  followed by all Zcash implementations.
+
+* A Standards Track ZIP describes any non-consensus change that affects
+  most or all Zcash implementations, such as a change to the network
+  protocol, or any change or addition that affects the interoperability
+  of applications using Zcash.
+
+Consensus and Standards Track ZIPs consist of two parts, a design document
+and a reference implementation.
+
+* An Informational ZIP describes Zcash design issues, or general
+  guidelines or information for the Zcash community, that do not fall
+  into either of the above categories. Informational ZIPs do not
+  necessarily represent a Zcash community consensus or recommendation,
+  so users and implementers are free to ignore Informational ZIPs or
+  follow their advice.
+
+* A Process ZIP describes a process surrounding Zcash, or proposes a
+  change to (or an event in) a process. Process ZIPs are like Standards
+  Track ZIPs but apply to areas other than the Zcash protocol itself.
+  They may propose an implementation, but not to Zcash's codebase; they
+  often require community consensus; unlike Informational ZIPs, they
+  are more than recommendations, and users are typically not free to
+  ignore them. Examples include procedures, guidelines, changes to the
+  decision-making process, and changes to the tools or environment used
+  in Zcash development.
+
+New categories may be added by consensus among the ZIP Editors.
+
+
+ZIP Status Field
+================
+
+* Draft: All initial ZIP submissions have this status.
+
+* Withdrawn: If the Champion decides to remove the ZIP from
+  consideration by the community, they may set the status to Withdrawn.
+
+* Active: Typically only used for Process/Informational ZIPs, achieved
+  once rough consensus is reached in PR/mailing list from Draft Process
+  ZIP.
+
+* Proposed: Typically the stage after Draft, added to a ZIP after
+  consideration and feedback from the community. The ZIP Editors must
```

Comment:
This text appears to indicate that the ZIP may not have reached rough consensus before reaching this stage, but in the Specification section below it is clear that rough consensus is required. Perhaps:

```suggestion
  consideration, feedback, and rough consensus from the community. The ZIP Editors must
```